CONTRIBUTING.md

```diff
@@ -87,6 +87,140 @@ This script will:
     - `index.js` - define your custom element here, by extending our provided JS classes.
     - `index.scss` - here goes your CSS.
 
+## Custom Elements
+
+We stick to the [Custom Elements V1 spec](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements).
+
+There are a few key principles you have to know:
+- Custom Elements are **asynchronous**, which means
+   - they only render if their definition (JS) is ready
+   - a child could render before it's parent
+   - this leads to **FOUC**
```

Comment:
i would not add shortcuts on a description or at least please add the meaning in brackets like:

FOUC (Flash of Unstyled Content)